### ISSUE TITLE
Options to bring back the splash overlay

### ISSUE THREAD

**Issue by Oondanomala** [ChachyDev/lazy-language-loader#10]
Please add an option to bring back the splash overlay and one to remove the search bar added in 0.2.6, i like having a clean vanilla like menu

**Comment by ChachyDev** [comment]
I'll consider this, just trying to think how to achieve this without, preferably, dependance on other mods

**Comment by seriousfreezing** [comment]
I totally agree about having a way to hide the language search even though it's useful, I wish I couldn't have it in my modpack :D

**Comment by ChachyDev** [comment]
I may take the same approach here as you mentioned in #15 about a simple text file to edit
